assets/app/styles/_flexbox.less

```diff
@@ -132,4 +132,182 @@
      -moz-align-self: @align;
       -ms-align-self: @align;
           align-self: @align;
-}
\ No newline at end of file
+}
+
+
+
+
+// Generic layout attr utils, inspired by Angular Material/Polymer flex layouts
```

Comment:
This looks very expressive. Should we use the different flex mixing above in the rules below to pick up the vendor-prefixed styles?